uniffi_bindgen/src/bindings/python/templates/macros.py

```diff
@@ -5,15 +5,14 @@
 #}
 
 {%- macro to_ffi_call(func) -%}
-
 rust_call_with_error(
     {%- match func.throws() -%}
     {%- when Some with (e) -%}
     {{ e|class_name_py }}
     {%- else -%}
     InternalError
-    {%- endmatch -%},
-    _UniFFILib.{{ func.ffi_func().name() }}{% if func.arguments().len() > 0 %},{% endif %}{% call _arg_list_ffi_call(func) -%}
+    {%- endmatch -%}
+    ,_UniFFILib.{{ func.ffi_func().name() }}{% if func.arguments().len() > 0 %},{% endif %}{% call _arg_list_ffi_call(func) -%}
```

Comment:
This little comma tweak makes the call emit entirely on one line, which seemed to help with some formatting issues in the generated code.